app/views/categories/show.html.haml

```diff
@@ -0,0 +1,7 @@
+%section.genre
+  %header.clearfix
+    %h1 #{@category.name}
+  %article.row
+    - @category.videos.each do |video|
+      .video.col-md-2
+        %img(src="#{video.small_cover_url}")
```

Comment:
You could eventually use `= image_tag video.small_cover_url` and it would do the same with less brackets.